### PR TITLE
Fix copy on registration pending view

### PR DESCRIPTION
**Pull request by yuki24**
addresses https://artsyproduct.atlassian.net/browse/AUCT-508

Just a quick copy update:

## Screenshot

Screen Shot 2019-08-01 at 5 06 14 PM

### CODE REVIEW

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx [hunk 1]
```diff
@@ -38,9 +38,9 @@ const registrationCompleteMessage = {
 const registrationPendingMessage = {
   title: "Registration pending",
   description:
-    "You will receive an email from an Artsy specialist once your registration has been confirmed.\n" +
+    "Artsy is reviewing your registration and you will receive an email when it has been confirmed. Please email specialist@artsy.net with any questions.\n" +
```
Comment (dblandin) [inline src/lib/Components/Bidding/Screens/RegistrationResult.tsx]: Do we usually wrap email addresses in a mailto link?

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx [hunk 2]
```diff
@@ -38,9 +38,9 @@ const registrationCompleteMessage = {
...
     "\n" +
-    "In the meantime, you can still view works and watch lots that you’re interested in.",
+    "In the meantime, you can still view works and watch lots you’re interested in.\n",
```
Comment (dblandin) [inline src/lib/Components/Bidding/Screens/RegistrationResult.tsx]: Is this new trailing new line for padding?

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx (hunk 1 above)
Comment (dleve123) [reply src/lib/Components/Bidding/Screens/RegistrationResult.tsx]: Considering that we use a [`mailto` for the registration failed component](https://github.com/artsy/emission/blob/04af01b03b463cda21e620f67b42a040e43f7556/src/lib/Components/Bidding/Screens/RegistrationResult.tsx#L48), I think it's safe to assume that we should do so here. Can you make that change @yuki24?

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx (hunk 2 above)
Comment (dleve123) [reply src/lib/Components/Bidding/Screens/RegistrationResult.tsx]: +1 to this question? Is this newline addition intentional?

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx (hunk 1 above)
Comment (yuki24) [reply src/lib/Components/Bidding/Screens/RegistrationResult.tsx]: Okay, the description can actually be markdown and it is very easy to make it a link:

Screen Shot 2019-08-02 at 10 51 50 AM

This link does not work on Emission due to an unhandled native event, but if I recall correctly this is handled properly in Eigen. We should QA this anwyay to make sure it works in the actual app.

**Comment by yuki24** [comment]
#mergeongreen

File: src/lib/Components/Bidding/Screens/RegistrationResult.tsx (hunk 1 above)
Comment (yuki24) [reply src/lib/Components/Bidding/Screens/RegistrationResult.tsx]: There seems to be a bug around the email click but there is another PR that addresses it: https://github.com/artsy/emission/pull/1769 so it's likely that it still crashes but it'll be fixed eventually.

**Pull request merged**